cmd/up/exec.go

```diff
@@ -440,7 +440,7 @@ func (up *upContext) RunCommand(ctx context.Context, cmd []string) error {
 			hybridCtx := &HybridExecCtx{
 				Dev:       up.Dev,
 				Name:      up.Manifest.Name,
-				Namespace: up.Manifest.Namespace,
+				Namespace: okteto.GetContext().Namespace,
```

Comment:
Same as before. We have in line 429 `up.Namespace`

Reply:
fixed in b262d22a36a539a60335bee8182f6a2c348d7642